src/main.rs

```diff
@@ -112,7 +114,8 @@ impl<'a> Printer<'a> {
 
     fn print_byte(&mut self, b: u8) -> io::Result<()> {
         if self.idx % 16 == 1 {
-            write!(self.stdout, "│ ");
+            let byte_str = format!("{:08x}", self.idx - 1);
+            write!(self.stdout, "│{}│ ", Byte(0x00).color().paint(byte_str))?;
```

Comment:
We could use `COLOR_NULL` here instead of `Byte(0x00).color()`. But I'd prefer to add a new constant for this (`COLOR_OFFSET`). It could be the same grey color as for NULL bytes, though.

Reply:
Sounds like a good idea! I've pushed a new fix. Let me know if there's anything else you want me to change.